skgstat/MetricSpace.py

```diff
@@ -379,10 +378,12 @@ def dists(self):
             self._dists = dists
         return self._dists
 
-# Subfunctions of RasterEquidistantMetricSpace
-
 def get_disk_sample(coords, center, center_radius, rnd_func, sample_count):
-
+    """
+    Subfunction for RasterEquidistantMetricSpace.
+    Calculates the indexes of a subsample in a disk "center sample".
+    Same parameters as in the class.
+    """
     # First index: preselect samples in a disk of certain radius
```

Comment:
Can these functions be used outside of the main class? If yes, it could make sense to document the arguments in the docstring quickly.

Also, is it possible to find meaningful defaults? At least for sample_count?

Reply:
Only used by the class, but they have to be outside to be pickled by multiprocessing.
I added a comment stating just that + added typing annotation for parameters + added a leading underscore to the function names so that they are not accessible by users importing MetricSpace. All in a5702c9